config/env_sample.json

```diff
@@ -176,13 +176,13 @@
             "app_display_name": "Canvas Files",
             "canvas_course_id_format": "LONG",
             "urls": {"prefix": "https://example.instructure.com/files/", "postfix": "/download?download_frd=1"}
         },
         "leccap": {
-            "query": "select 'leccap' AS resource_type, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'), 48) AS STRING) AS resource_id, @canvas_data_id_increment + CAST(JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_user_id') AS INT64) AS user_id, @canvas_data_id_increment + CAST(JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_course_id') AS INT64) AS course_id, JSON_EXTRACT_SCALAR(event, '$.object.name') as name, datetime(EVENT_TIME) as access_time FROM event_store.events where JSON_EXTRACT_SCALAR(event, '$.edApp') = 'https://leccap.engin.umich.edu/#applicationName=Lecture+Capture' and type = 'MediaEvent' and JSON_EXTRACT_SCALAR(event, '$.action') = 'Started' and JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_course_id') is not null and JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_course_id') IN UNNEST(@course_ids_short)",
+            "query": "select 'leccap' AS resource_type, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'), 48) AS STRING) AS resource_id, @canvas_data_id_increment + CAST(JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_user_id') AS INT64) AS user_id, @canvas_data_id_increment + CAST(JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_course_id') AS INT64) AS course_id, JSON_EXTRACT_SCALAR(event, '$.object.name') as name, datetime(EVENT_TIME) as access_time FROM event_store.events where JSON_EXTRACT_SCALAR(event, '$.edApp') = 'https://leccap.engin.umich.edu/#applicationName=Lecture+Capture' and type = 'MediaEvent' and JSON_EXTRACT_SCALAR(event, '$.action') = 'Started' and JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_course_id') is not null and JSON_EXTRACT_SCALAR(event, '$.federatedSession.messageParameters.custom_canvas_course_id') IN UNNEST(@course_ids_short) and event_time > @course_start_time",
```

Comment:
It would be great to see these queries re-formatted for better readability!

Reply:
I don't believe JSON allows for multiline strings. It would have to be some hacky solution where we maybe had an array that had each pice on a different line then the code combined it. :/ 

I think these queries can be simplified in other ways though by pulling the patterns out of them but that's a fix I haven't gotten to yet.

For example:
```
SELECT 'canvas' AS resource_type, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'), 35) AS STRING) AS resource_id, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.membership.member.id'), 29) AS INT64) AS user_id, CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31) AS INT64) AS course_id, COALESCE(JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].asset_name'), JSON_EXTRACT_SCALAR(event, '$.object.extensions[\\'com.instructure.canvas\\'].filename'), JSON_EXTRACT_SCALAR(event, '$.object.name'), 'attachment') as name, datetime(EVENT_TIME) as access_time`
````
Could probably be something like
```
resource_type: "canvas",
resource_id: "SUBSTR(JSON_EXTRACT_SCALAR(event, '$.object.id'), 35)",
user_id: "SUBSTR(JSON_EXTRACT_SCALAR(event, '$.membership.member.id'), 29)",
course_id: "SUBSTR(JSON_EXTRACT_SCALAR(event, '$.group.id'),31)",
```

This could probably be improved further since many of these are pulling substrings out of the json event so that could be special too. 

We could probably improve on all other parts of this query too by finding and replacing the patterns, or using some "methods" to do that. I really don't like how this is either but this is the real query. ;)

Reply:
I have hacked the way via turning the long sql statement into array (multi lined) and the array items are joined later to form one string again.

As for the query simplification, I think it is worth another issue (issue #856)